packages/devtools/devtools-extension/src/devtools.ts

```diff
@@ -18,7 +16,6 @@ const createPanel = async () => {
   }
 
   log('Attempting to create panel...');
-  await waitForDXOS();
```

Comment:
It was breaking the extension (something about using `eval` before we have a panel is causing the error), and we are waiting for the hook in `panel.ts`, so it is fine to delete it here.

Reply:
That's funny that that is now what's breaking it. Before this is what helped make it more stable by ensuring that it didn't try to initialize a client before it could connect.